src/app/static/src/js/utils/types.ts

```diff
@@ -137,7 +137,7 @@ export interface WorkflowReportWithDependencies {
     depends_on?: string[]
 }
 
-export interface WorkflowSummary {
+export interface WorkflowSummaryEndpoint {
```

Comment:
I'm not really on board with this rename tbh.. since it's not an endpoint in any sense.. I can see that there's another WorkflowSummary that you want to distinguish this from - that's the one that get used in WorkflowProgress right? It's for workflow that have already started. 
Rather than 'Endpoint' I think I'd at least be happier with either calling this one "WorkflowSummaryResponse" or calling the other one "WorkflowRunSummary" - or both.